### ISSUE TITLE
Errors

### ISSUE THREAD

**Issue by FRex** [timknip/pycsg#8]
I want to have a single cube and then from it subtract another one that is rotated along around three axes. However it results in errors, the outside cube has a slash through it (so to speak).
See (textures being messy is okay, the hole is what the mistake is): http://i.imgur.com/HxpoRCX.png

The second cube is defined in the txt file.
Format of the txt is:
int amount of meshes,
for each mesh: int operation () + 0 0 0 (quirk of the thing that spits these out) + int amount of polys,
for each poly format is: amount of vertices + that many triplets of x y z.

It's just two cubes with 256 unit long sides but each vertex was transformed by a 4x4 matrix.
If I use the same cube with no rotations (identity matrix or just a translation) it works.
Is this an error related to numeric precision, some csg quirk?

Here is the data that causes the error (there are two cubes but first is skipped because it has op 0) and the Python3 script I use:
https://gist.github.com/FRex/6ca78af50f21e1cf7bdd342965a2b6f4

**Comment by themmes** [comment]
@FRex Did you ever resolve your problem? Are you still using this library, or have you found an alternative?

**Comment by FRex** [comment]
@themmes Sorry, no to all. It was one of my random playthings with Irrlicht and CSG (I have a thing for retro games and DarkEngine and Unreal Engine 1 were almost all about CSG for level geometry), not anything serious.

**Comment by themmes** [comment]
@FRex I see, thanks for your quick reply. It seems impossible to find a capable 3D geometry library for Python..

**Comment by FRex** [comment]
@themmes Yes, I tried hard to find a C, C++ or Python 3D CSG library that wasn't abandoned, old, with bugs, inconvenient to build (requiring GMP), etc. too back then while for 2D it was easy to find the equivalent of CSG (ClipperLib C++/C#/Delphi by Angus Johnson).
